spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe " Ingredient index page" do
+  before(:each) do 
+    @f_alfredo = Recipe.create!(name: "Fetuccini Alfredo", complexity: 3, genre: "Italian")
+    @chili = Recipe.create!(name: "Chili", complexity: 2, genre: "American")
+    @curry = Recipe.create!(name: "Curry", complexity: 3, genre: "Indian")
+
+    @pasta = Ingredient.create!(name:"Pasta", cost: 2)
+    @pasta_sauce = Ingredient.create!(name:"Pasta Sauce", cost: 2)
+    @parmesan = Ingredient.create!(name:"Parmesan", cost: 5)
+    @mushrooms = Ingredient.create!(name:"Mushrooms", cost: 2)
+    @salt = Ingredient.create!(name:"Salt", cost: 1)
+    @beans = Ingredient.create!(name:"Beans", cost: 2)
+    @tomatoes = Ingredient.create!(name:"Tomatoes", cost: 2)
+    @broth = Ingredient.create!(name:"Broth", cost: 2)
+    @chicken = Ingredient.create!(name:"Chicken", cost: 7)
+    @curry_paste = Ingredient.create!(name:"Curry Paste", cost: 2)
+    @coconut_milk = Ingredient.create!(name:"Coconut Milk", cost: 2)
+    @sweet_potatoes = Ingredient.create!(name:"Sweet Potatoes", cost: 2)
+  end
+
+    it "has an ingredient index page" do
+
+      #User Story 1 - Ingredients Index
+      # As a visitor,
+      # When I visit '/ingredients'
+      # I see a list of all the ingredients including their name and cost
+      # (e.g. "Ground Beef: 2"
+      #      "Salt: 4")
+      visit "/ingredients"
+
+      expect(page).to have_content("Chicken")
+      expect(page).to have_content("Pasta")
+      expect(page).to have_content("Pasta Sauce")
+      expect(page).to have_content("Salt")
+      expect(page).to have_content("Mushrooms")
+      expect(page).to have_content("Sweet Potatoes")
+      expect(page).to have_content(5)
+      expect(page).to have_content(7)
+      expect(page).to have_content(2)
```

Comment:
These assertions are good, but could be more robust if you anchored them to text they should appear next to. You've set up your view well with labels next to each of these attributes, so you could change your assertions to     `expect(page).to have_content("Cost: #{pasta.cost}")`

This will make sure that this data is located in the correct place, and is meaningfully rendered on the page. Also, if the cost were `7` and there happened to be a footer on the page that happened to include a 7, you could get a false positive by just asserting for the presence of `7`. So it's better to be specific about where the 7 appears and assert the presence of `Cost: 7`